core/store/LevelDBStore/LevelDBStore.go

```diff
@@ -649,7 +619,7 @@ func (bd *LevelDBStore) SaveBlock(b *Block, ledger *Ledger) error {
 	}
 
 	log.Info("len(bd.header_index) is ", len(bd.header_index), " ,b.Blockdata.Height is ", b.Blockdata.Height)
-	if b.Blockdata.Height-uint32(len(bd.header_index)) >= 1 {
+	if b.Blockdata.Height >= uint32(len(bd.header_index)) + 1 {
```

Comment:
how about add the bracket which more clear,  e.g:

b.Blockdata.Height >= (uint32(len(bd.header_index)) + 1)